phpBB/phpbb/di/container_builder.php

```diff
@@ -540,6 +559,16 @@ protected function get_container_filename()
 	}
 
 	/**
+	 * Get the filename under which the dumped extensions autoloader will be stored.
+	 *
+	 * @return string Path for dumped extensions autoloader
+	 */
+	protected function get_autoload_filename()
+	{
+		return $this->get_cache_dir() . 'autoload_' . md5($this->phpbb_root_path) . '.' . $this->php_ext;
```

Comment:
well, it's not necessary (it will always be erased by the container builder) but I can add it